### PR TITLE
Admin actions

### PR DESCRIPTION
**Pull request by jsed500**

### CODE REVIEW

File: components/clinical/admin-actions/src/atoms/admin-action-description.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { FC } from 'react'
+import styled from '@emotion/styled'
+import { Maybe } from '@ltht-react/types'
+
+const StyledDescription = styled.div`
```
Comment (Moza805) [inline components/clinical/admin-actions/src/atoms/admin-action-description.tsx:5]: For consistency, would rather either
a) inline this into `admin-action.tsx` with the rest of the components
b) extract other emotion styled components form `admin-action.tsx` into their own component files

File: components/clinical/admin-actions/src/atoms/admin-action-description.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  flex: 1;
+`
+
+const Description: FC<IProps> = ({ description }) => {
+  if (description) return <StyledDescription>{description}</StyledDescription>
```
Comment (Moza805) [inline components/clinical/admin-actions/src/atoms/admin-action-description.tsx:10]: React standard would be to use children prop here

File: components/clinical/admin-actions/src/organisms/admin-action.tsx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+          value={!adminAction.isLoading ? 'Perform Action' : 'Loading'}
+          icon={adminAction.isLoading && <SpinnerIcon size="medium" />}
+          onClick={() => actionClickHandler(adminAction)}
+        />
+      )}
+      {adminAction.isSuccess === true && (
+        <SuccessButton type="button" value="Done" icon={<CheckIcon size="medium" />} />
+      )}
+      {adminAction.isSuccess === false && (
+        <FailButton type="button" value="Failed - Please report to tech services" icon={<CrossIcon size="medium" />} />
```
Comment (Moza805) [inline components/clinical/admin-actions/src/organisms/admin-action.tsx:53]: This language is probably too specific to ltht to make it into an open source repo?

**Review by Moza805 — CHANGES_REQUESTED** [review]

File: components/clinical/admin-actions/src/organisms/admin-action.tsx (hunk 3 above)
Comment (jsed500) [reply components/clinical/admin-actions/src/organisms/admin-action.tsx:53]: Thats true! will abstract out

**Review by Moza805 — APPROVED** [review]

**Pull request merged**